bin/electron-osx-sign-usage.txt

```diff
@@ -35,6 +35,9 @@ DESCRIPTION
     The keychain name.
     Default to system default keychain.
 
+  --no-getekeeper-check
```

Comment:
Like `--pre-auto-entitlements, --no-pre-auto-entitlements`, `--gatekeeper-assess, --no-gatekeeper-assess` may then enable/disable the feature.